app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> None:
```

Comment:
Specify inner type of list
```suggestion
def calculate_team_total_rating(team: list[Player]) -> None:
```